execution_engine/src/storage/trie/mod.rs

```diff
@@ -450,6 +450,31 @@ impl Display for TrieOrChunkId {
     }
 }
 
+impl ToBytes for TrieOrChunkId {
+    fn to_bytes(&self) -> Result<Vec<u8>, bytesrepr::Error> {
+        let mut buf = bytesrepr::allocate_buffer(self)?;
+        self.write_bytes(&mut buf)?;
+        Ok(buf)
+    }
+
+    fn serialized_length(&self) -> usize {
+        ToBytes::serialized_length(&self.0) + ToBytes::serialized_length(&self.1)
```

Comment:
It's a bit more canonical to write `self.0.serialized_length()`.  Same comment applies to a few such instances in this PR.